lib/rex/ui/text/input/readline.rb

```diff
@@ -187,6 +188,19 @@ def readline_with_output(prompt, add_history=false)
       end
     end
 
+    private
+
+    def wrap_tab_compete_proc(proc)
```

Comment:
I wonder if there's a name that better captures the fact this is for error handling purposes?
`wrap_tab_compete_proc` is somewhat abstract. Maybe `with_error_handling(tab_complete_proc)` 🤔 

```suggestion
    def with_error_handling(proc)
```